### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by VoitecP**
- Added  Instalation manual in Readme  file

Personal Note:
This task it is good  as a playgroud to train Git, so please correct me if I will do something wrong

### CODE REVIEW

**Comment by jacoor** [comment]
@VoitecP 
Couple remarks
1. Please change the branch you are merging to, to "dev" not master. Master was automatically suggested by github and should be changed manually. You can use "edit" option near PR title. But, before doing so, please check how many files are changed here - 12. So this is clear indication of something wrong. 
2. Please read: https://github.com/akademiait/Dshop/blob/dev/README.md
3. When creating pull request please request review from fellow team members:
image
on far right, click the "cog icon" and pick Dshop team. 

Thanks!

File: README.md [hunk 1]
```diff
@@ -16,3 +16,35 @@ Good reading on working with code changes and pull request is https://google.git
...
+
+Clone repository to specyfic folder (ex. Dshop):
+```
+git clone https://github.com/akademiait/Dshop.git
+```
+You need to have installed Poetry package. If you don't have, please install using this command:
+```
+pip install poetry
+```
+Navigate to Dshop folder by command:
```
Comment (jacoor) [inline README.md:30]: A constructive comment.

File: README.md [hunk 2]
```diff
@@ -16,3 +16,35 @@ Good reading on working with code changes and pull request is https://google.git
...
+```
+cd ./Dshop
+```
+Set poetry global option, to use project folder as place to hold Virtual enviroment (recomended)
+```
+poetry config virtualenvs.in-project true
```
Comment (jacoor) [inline README.md:36]: Another constructive comment

File: README.md [hunk 3]
```diff
@@ -16,3 +16,35 @@ Good reading on working with code changes and pull request is https://google.git
 5. At least one approval by other team member is required before pull request can be merged.
 6. After pull request is approved and code quality + tests are passed, pull request is merged by the author.
 7. It is author responsibility to watch over pull request, bump if there is no code review done, fix issues and merge pull request.
+
+#### How to Setup
+
+Clone repository to specyfic folder (ex. Dshop):
+```
+git clone https://github.com/akademiait/Dshop.git
+```
```
Comment (jacoor) [inline README.md:25]: Testing couple lines

File: README.md [hunk 4]
```diff
@@ -16,3 +16,35 @@ Good reading on working with code changes and pull request is https://google.git
...
+Navigate to Dshop folder by command:
+```
+cd ./Dshop
+```
+Set poetry global option, to use project folder as place to hold Virtual enviroment (recomended)
+```
+poetry config virtualenvs.in-project true
+```
+Install virtual enviroment, using current dependencies
```
Comment (jacoor) [inline README.md]:
```suggestion
Install virtual enviroment, using current dependencies.
```

File: README.md (hunk 1 above)
Comment (VoitecP) [reply README.md:30]: thank you

File: README.md (hunk 2 above)
Comment (VoitecP) [reply README.md:36]: great

**Review by jacoor — APPROVED** [review]

**Pull request merged**